locale/en_US/user.po

```diff
@@ -128,6 +128,9 @@ msgstr "Thank you for activating your account. You may now log in using the cred
 msgid "user.login.changePasswordInstructions"
 msgstr "You must choose a new password before you can log in to this site.<br /><br />Please enter your username and your current and new passwords below to change the password for your account."
 
+msgid "user.login.resetPasswordInstructions"
+msgstr "Enter new password to update login password."
```

Comment:
Because this is going into the `stable-3_3_0` branch, and translation tools are working on `main`, it's likely that a lot of end users will encounter `##untranslated.locale.keys##` after this change is introduced. I'd recommend using Google Translate to come up with provisional translations for all the languages that are above a threshold of completeness, and flagging the added translations as `#fuzzy` so that they can be reviewed/fixed.